static/locales/en-US.yaml

```diff
@@ -187,35 +187,32 @@ Settings:
       720p: 720p
       1080p: 1080p
       1440p: 1440p
-      4k: 4k
-      8k: 8k
+      4k: 4K
+      8k: 8K
   Privacy Settings:
     Privacy Settings: Privacy Settings
     Remember History: Remember History
     Save Watched Progress: Save Watched Progress
     Clear Search Cache: Clear Search Cache
-    Are you sure you want to clear out your search cache?: Are you sure you want to
-      clear out your search cache?
-    Search cache has been cleared: Search cache has been cleared
+    Are you sure you want to clear out your search cache?: Clear search cache?
+    Search cache has been cleared: Search cache cleared
     Remove Watch History: Remove Watch History
-    Are you sure you want to remove your entire watch history?: Are you sure you want
-      to remove your entire watch history?
-    Watch history has been cleared: Watch history has been cleared
+    Are you sure you want to remove your entire watch history?: Remove entire watch history?
+    Watch history has been cleared: Watch history cleared
     Remove All Subscriptions / Profiles: Remove All Subscriptions / Profiles
-    Are you sure you want to remove all subscriptions and profiles?  This cannot be undone.: Are
-      you sure you want to remove all subscriptions and profiles?  This cannot be
-      undone.
+    Are you sure you want to remove all subscriptions and profiles?  This cannot be undone.: Remove
+  all subscriptions and profiles forever?
   Subscription Settings:
     Subscription Settings: Subscription Settings
     Hide Videos on Watch: Hide Videos on Watch
     Fetch Feeds from RSS: Fetch Feeds from RSS
     Manage Subscriptions: Manage Subscriptions
   Distraction Free Settings:
-    Distraction Free Settings: Distraction Free Settings
+    Distraction Free Settings: Distraction-Free Settings
     Hide Video Views: Hide Video Views
-    Hide Video Likes And Dislikes: Hide Video Likes And Dislikes
+    Hide Video Likes And Dislikes: Hide Video 👍 / 👎
```

Comment:
The intention is good, although the consequences are not that great. There are are people who might be color blind or have problems with emoticons, so simple text will have to stay here.

Reply:
How does colour-blindness factor into seeing outlines of shapes? YouTube uses it.

Reply:
Yes, but it might be a problem if no alternative is added.
Here a few resources. Additionally we had a request from someone to make the app usable with screen readers. Unfortunately we were not able to implement that yet due to the many changes that happen all the time. But here are some resources to read on. The last one is behind a paywall bc it is a scientific paper, but the abstract is still available:

https://www.ashleysheridan.co.uk/blog/Emoji+and+Accessibility
https://uxdesign.cc/accessibility-matters-632fa2fb0672
https://dl.acm.org/doi/10.1145/3313831.3376267